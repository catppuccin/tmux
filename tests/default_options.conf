# vim:set ft=tmux:

# Tests that the default options are set correctly
source -F "#{d:current_file}/../catppuccin_options_tmux.conf"

show -g @catppuccin_flavor
