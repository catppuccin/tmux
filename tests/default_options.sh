#!/usr/bin/env bash

script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
# shellcheck disable=SC1091
source "${script_dir}/helpers.sh"

# Tests that the default options are set correctly
tmux source "${script_dir}/../catppuccin_options_tmux.conf"
tmux source "${script_dir}/../catppuccin_tmux.conf"

print_option @catppuccin_flavor
print_option @catppuccin_menu_selected_style
print_option @catppuccin_pane_active_border_style
