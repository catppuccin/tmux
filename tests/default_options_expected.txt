@catppuccin_flavor mocha
@catppuccin_menu_selected_style fg=#{@thm_surface_0},bg=#{@thm_yellow}
@catppuccin_pane_active_border_style ##{?pane_in_mode,fg=#{@thm_lavender},##{?pane_synchronized,fg=#{@thm_mauve},fg=#{@thm_lavender}}}
